packages/devtools-connect/package.json

```diff
@@ -0,0 +1,94 @@
+{
+  "name": "@mongodb-js/devtools-connect",
+  "version": "2.4.3",
+  "description": "A connection establishment utility for MongoDB developer tools",
+  "homepage": "https://github.com/mongodb-js/devtools-connect",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/mongodb-js/devtools-connect.git"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "bugs": {
+    "url": "https://github.com/mongodb-js/devtools-connect/issues"
+  },
+  "main": "dist/index.js",
+  "exports": {
+    "require": "./dist/index.js",
+    "import": "./.esm-wrapper.mjs"
+  },
+  "files": [
+    "LICENSE",
+    "dist",
+    "package.json",
+    "README.md",
+    ".esm-wrapper.mjs"
+  ],
+  "scripts": {
+    "bootstrap": "npm run compile",
+    "prepublishOnly": "npm run compile",
+    "compile": "npm run compile-ts && gen-esm-wrapper . ./.esm-wrapper.mjs",
+    "typecheck": "tsc --noEmit",
+    "eslint": "eslint src/**/*.ts",
+    "prettier": "prettier",
+    "lint": "npm run eslint . && npm run prettier -- --check .",
+    "depcheck": "depcheck",
+    "check": "npm run typecheck && npm run lint && npm run depcheck",
+    "check-ci": "npm run check",
+    "testonly": "nyc mocha --colors -r ts-node/register src/**/*.spec.ts",
+    "test": "npm run lint && npm run compile && npm run testonly",
+    "test-cov": "nyc -x \"**/*.spec.*\" --reporter=lcov --reporter=text --reporter=html npm run test",
+    "test-watch": "npm run test -- --watch",
+    "test-ci": "npm run test-cov",
+    "reformat": "npm run prettier -- --write .",
+    "prepack": "npm run compile",
+    "compile-ts": "tsc -p tsconfig.json"
+  },
+  "license": "Apache-2.0",
+  "dependencies": {
+    "@mongodb-js/oidc-http-server-pages": "0.1.0",
+    "lodash.merge": "^4.6.2",
+    "mongodb-connection-string-url": "^3.0.0",
+    "system-ca": "^1.0.2",
+    "mongodb": "^5.8.1 || ^6.0.0"
```

Comment:
You're running into the problem with `saslprep`, right? You can just add saslprep itself as a `devDependencies` entry here to solve that (we probably don't want to directly make `mongodb` a dependency of `devtools-connect`, just a peer dependency like it previously was)

Reply:
oh, thank you!